x-pack/platform/plugins/shared/fleet/server/services/epm/packages/rollback.ts

```diff
@@ -61,6 +62,19 @@ export async function rollbackInstallation(options: {
   );
   const packagePolicySOs = packagePolicySORes.saved_objects;
 
+  const managedRollbackError = new PackageRollbackError(
+    `Cannot rollback integration with managed package policies`
+  );
+  if (packagePolicySOs.some((so) => so.attributes.is_managed)) {
+    throw managedRollbackError;
+  }
+  // checking is_managed flag on agent policy, it is not always set on package policy
+  const agentPolicyIds = uniq(packagePolicySOs.flatMap((so) => so.attributes.policy_ids ?? []));
+  const agentPolicies = await agentPolicyService.getByIds(savedObjectsClient, agentPolicyIds);
```

Comment:
Here we probably want to search accross all spaces

```
agentPolicyService.getByIds(savedObjectsClient, [{ id: policy1, spaceId: '*'}]);
```
Related to space should we use `getInternalUserSOClientWithoutSpaceExtension` as a `savedObjectsClient`  to be sure this work accross all spaces>